pkg/cmd/pulumi/auth/login.go

```diff
@@ -153,15 +153,18 @@ func NewLoginCmd() *cobra.Command {
 					return fmt.Errorf("could not determine current cloud: %w", err)
 				}
 			} else if url := strings.TrimPrefix(strings.TrimPrefix(
-				cloudURL, "https://"), "http://"); strings.HasPrefix(url, "app.pulumi.com/") ||
-				strings.HasPrefix(url, "pulumi.com") {
+				cloudURL, "https://"), "http://"); strings.HasPrefix(url, "pulumi.com") {
```

Comment:
Wait, no, this is worse now.  We always error out when there's a prefix of `pulumi.com` now, which is wrong.  I think this whole check should go away now.